__device-tests__/gutenberg-editor-rotatation.test.js

```diff
@@ -52,6 +52,9 @@ describe( 'Gutenberg Editor tests', () => {
 		await editorPage.sendTextToParagraphBlock( paragraphBlockElement, testData.mediumText );
 
 		await toggleOrientation( driver );
+		if ( isAndroid() ) {
+			await driver.hideDeviceKeyboard();
```

Comment:
Let's add a comment here about why we need to do this, especially on Android.

We'll actually want to remove that requirement down the road by not having the RichText component trigger the extracted EditText UI (`flagNoExtractUi` and friends). But for now, adding a line comment and a "TODO" marker should be OK. WDYT @JavonDavis ?

Reply:
Sounds good I'll add that! 👍🏽